airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -206,13 +206,14 @@ public static void main(String[] args) throws Exception {
     final JobPersistence jobPersistence = new DefaultJobPersistence(database);
 
     final String airbyteVersion = configs.getAirbyteVersion();
-    final Optional<String> airbyteDatabaseVersion = jobPersistence.getVersion();
-    if (airbyteDatabaseVersion.isEmpty()) {
+    final Optional<String> checkAirbyteInitDBVersion = jobPersistence.getVersion();
+    if (checkAirbyteInitDBVersion.isEmpty()) {
```

Comment:
```suggestion
    if (jobPersistence.getVersion().isEmpty()) {
```

The difference between `checkAirbyteInitDBVersion` and `airbyteDatabaseVersion` would be less confusing if we just inlined here and made `checkAirbyteInitDBVersion` unnecessary.